extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -387,6 +368,9 @@ export class VisibilityModel {
   isVisibilityMatch_(visibility) {
     dev().assert(visibility >= 0 && visibility <= 1,
         'invalid visibility value: %s', visibility);
+    if (this.spec_.visiblePercentageMin == 1) {
```

Comment:
Without this, if they set min=max=100 then we would never fire.
This opens up the discussion of should we just change ">" on line 374 to ">=" but in that case we would need to handle the special case of min=0 since visibility will always be >=0.

Reply:
do we need to handle this case? i would consider it as a mis-config.

Reply:
Without this special case, we don't provide any way to say "tell me when the add is 100% viewable". I feel that we should allow that in some way...whether it is this particular way or not isn't as important though.

Reply:
i see. we might treat 0 and 100 differently.
can you move it to a separate PR for more discussion?

Reply:
(...and if we choose a different implementation but no change to behaviour, I would also prefer to do it in a different PR)

Reply:
Sure, will move to separate PR. Thanks!